ruleset/README.md

```diff
@@ -0,0 +1,48 @@
+# Wazuh Ruleset
+
+[![Slack](https://img.shields.io/badge/slack-join-blue.svg)](https://goo.gl/forms/M2AoZC4b2R9A9Zy12)
+[![Email](https://img.shields.io/badge/email-join-blue.svg)](https://groups.google.com/forum/#!forum/wazuh)
+[![Documentation](https://img.shields.io/badge/docs-view-green.svg)](https://documentation.wazuh.com)
+[![Documentation](https://img.shields.io/badge/web-view-green.svg)](https://wazuh.com)
+
+Wazuh ruleset is used to detect attacks, intrusions, software misuse, configuration problems, application errors, malware, rootkits, system anomalies or security policy violations.
+
+The ruleset includes compliance mapping with PCI DSS v3.1 and CIS.
+
+## Installation
```

Comment:
I would remove the "Installation" section, there is no installation nor update process related to the ruleset now.

Reply:
+1

Reply:
You are right! Fixed.